docs/Makefile

```
@@ -39,7 +43,14 @@ linkcheck:
 
 woke:
 	type woke >/dev/null 2>&1 || { snap install woke; exit 1; }
-	woke *.rst **/*.rst -c https://github.com/canonical-web-and-design/Inclusive-naming/raw/main/config.yml
+
+	# Woke appears to have a issue where it skips .*ignore files unless it is executed from the repo root.
+	# This is mitigated by changing to the repo root and running woke on the relative docs path.
+	# Tested on 19/07/24 with woke 0.19.0-4-g5d52c15
+	cd ..; woke $(RELCWD)/**/*.rst $(RELCWD)/*.rst \
```

```suggestion
	cd ${REPOROOT}; woke $(RELCWD)/**/*.rst $(RELCWD)/*.rst \
```
that's safer perhaps?

---

I like it. That is more explicit as well. I pushed the change.